packages/react/radio-group/src/RadioGroup.tsx

```diff
@@ -61,14 +62,15 @@ const RadioGroup = React.forwardRef<RadioGroupElement, RadioGroupProps>(
       value: valueProp,
       required = false,
       orientation,
-      dir = 'ltr',
+      dir,
```

Comment:
that's different from the other place now, no double `||`, but equally removed the default too, I'm curious why.